src/v2/Apps/Consign/Routes/SubmissionFlow/UploadPhotos/__tests__/UploadPhotos.jest.tsx

```diff
@@ -94,11 +87,13 @@ describe("UploadPhotos", () => {
     jest.spyOn(global, "FileReader").mockImplementation(function () {
       this.readAsDataURL = jest.fn()
     })
+    ;(uploadPhoto as jest.Mock).mockResolvedValue("s3key")
```

Comment:
To improve readability a bit, this can be defined up above and then accessed via the variable:
```
const mockUploadPhoto = uploadPhoto as jest.Mock
```
(ditto elsewhere in PR)